### DISCUSSION TITLE
create a cartoon pixar style with Fooocus

### DISCUSSION THREAD

**Discussion by VENONUSA** [lllyasviel/Fooocus#3657] [Q&A] [answered]
Hi everyone, thanks for your support! Is it possible to create a Pixar-style or cartoon character with Fooocus? I've been trying, but the results aren't great 😭😭😭 Attached are 2 images that I would like to be able to reproduce with Fooocus.

![2](https://github.com/user-attachments/assets/acfa71f2-e905-4fbf-a328-31c7d4c57ee8)
![1](https://github.com/user-attachments/assets/8cf37d20-9878-4fb7-bf14-67944b851585)

**Comment by tryinghard247** [comment, marked as answer]
get example from civit ai . and copy the all source you need such checkpoint , lora , prompt , negative prompt . everything . after than if the result are closer like what you want you can change the setting again .
